neqo-transport/src/recovery.rs

```diff
@@ -640,8 +640,9 @@ impl LossRecovery {
         ack_delay: Duration,
     ) {
         let confirmed = self.confirmed_time.map_or(false, |t| t < send_time);
-        let sample = now - send_time;
-        rtt.update(&mut self.qlog, sample, ack_delay, confirmed, now);
+        if now >= send_time {
+            rtt.update(&mut self.qlog, now - send_time, ack_delay, confirmed, now);
```

Comment:
I think that this is more idiomatic.

```suggestion
        if Some(sample) = now.checked_sub(send_time) {
            rtt.update(&mut self.qlog, sample, ack_delay, confirmed, now);
```

Reply:
I have tried that, but it does not work checked_sub needs Duration; the output of checked_sub is Instant, but update() needs Duration.

Reply:
My mistake, I should have checked types.  What I meant was:

```suggestion
        if Some(sample) = now.checked_duration_since(send_time) {
            rtt.update(&mut self.qlog, sample, ack_delay, confirmed, now);
```

That is in rust 1.39, so it available in our minimum version.